src/dataflow-types/src/client/controller.rs

```diff
@@ -131,3 +128,15 @@ impl<C> Controller<C> {
         }
     }
 }
+
+/// Compaction policies for collections maintained by `Controller`.
+pub enum ReadPolicy<T = mz_repr::Timestamp> {
```

Comment:
I think you can leave out `= mz_repr::Timestamp` and nothing complains?

Reply:
You are right, and it even would have saved me one round of errors where I forgot the argument and it defaulted to this, despite needing to be generic.